mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -185,7 +185,7 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 		ARG 1 attribute
 	METHOD method_5997 attackLivingEntity (Lnet/minecraft/class_1309;)V
 		ARG 1 target
-	METHOD method_5998 getStackInHand (Lnet/minecraft/class_1268;)Lnet/minecraft/class_1799;
+	METHOD method_5998 getStack (Lnet/minecraft/class_1268;)Lnet/minecraft/class_1799;
```

Comment:
The removal of `inHand` seems to imply that this method can get stacks from more than just the two hands. Even with the `hand` parameter, I think it's better to leave the additional context.

Reply:
How does it imply that? `Hand` is an enum of only two values, it's not like it is an ambiguous `int` index.